### DISCUSSION TITLE
Change rate in Data Graph incorrect

### DISCUSSION THREAD

**Discussion by yellyfish1** [jomjol/AI-on-the-edge-device#2812] [General]
I'm using 	Release: v15.3.0 (Commit: 3fbff0a)
 Looking at the Data Graph I see the counter value at the right side with the  correct values. At the right side the change value is displayed. I can't correlate these values to the counter values. Please see attached screenshot.
![Data graph](https://github.com/jomjol/AI-on-the-edge-device-docs/assets/152398940/e1223151-2d6b-488b-a8dc-9f30f063f3bc)

**Comment by jomjol** [comment]
I do not understand the problem from your description.
The green line is the absolute counter readout (left scala, most probably in m³ if it's a water meter). The light blue bars show the change rate (right scala).
When ever the green line increases, there are blue bars - as expected.

Maybe you can mark the problem in the graph.

**Comment by yellyfish1** [comment]
Thanks for the response. Please find an addtional graph with some markers to explain the issue.
![Data graph2](https://github.com/jomjol/AI-on-the-edge-device/assets/152398940/3c8bc1e9-bccb-4fd0-ae19-4f1be4b678ec)

**Comment by jomjol** [comment]
Okay - now got your point. 

The change rate on the right is in volume / seconds!

So a reading of 0.004 means 0.004m³/s = 4 l/s ~ 240 l/min. Does that fit better?

**Comment by yellyfish1** [comment]
This make thing a little bit more clear, but a flow of 4l/s is not very likely. That's half a bucket in a second. I haven't seen places yet with so much pressure :). Comparing a change rate in seconds with a 5 minute interval measurement will always be very inaccurate. The outcome depends very much where the flow starts or ends within the 5 minute interval. With a continue flow for 10 minutes, one can have 3 different rates over three intervals.

I think it makes more sense to display at the change rate the absolute volume that passesd in an time interval. Is that possible?

**Comment by jomjol** [comment]
Got your point, for me the other information is more relevant. At the moment the graph is not configurable and it is only planned as a quick check. All relevant data analysis is done within data base tools.

**Comment by yellyfish1** [comment]
Ok. That is of course also possible. Thanks for the explaination anyway.
